app/services/recordService.py

```
@@ -236,19 +236,57 @@ def get_latest_records(db: Session, user_id: int) -> List[Record]:
+
+    # 최대 개수 검증
+    vrng(total_count <= 5, "기록 하나당 최대 5개의 회차만 생성할 수 있습니다.")
+
+    for exchange_data in exchange_list:
+
+        # 다음 회차 번호를 계산하고 할당
+        ex_no = _next_exchange_no(rec)
+
+        # 회차 번호가 이미 존재하는지 다시 검사합니다.
+        if find_exchange(rec, ex_no) is not None:
+            raise HTTPException(status_code=409, detail=f"회차 번호 충돌 발생.")
```

Ruff (0.14.5)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)